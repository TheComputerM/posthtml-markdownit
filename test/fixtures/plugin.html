<md>
    You can use emojis :)
</md>