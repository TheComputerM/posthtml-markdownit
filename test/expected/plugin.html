<p>You can use emojis ğŸ˜ƒ</p>
